pages/management/devices.md

```diff
@@ -32,6 +34,7 @@ An extremely useful feature of the devices detail page is the log window. This i
 
 On the device's Actions page we give you access to a number of useful per-device actions.
 
+<!-- NOTE: is it really necessary to reload the page while the ID is located on the same page?-->
```

Comment:
+1 for anchor-only. it's also resilient to the page being moved around because it only depends on the doc's internal structure